src/control/controlbehavior.cpp

```diff
@@ -161,19 +171,19 @@ double ControlAudioTaperPotBehavior::valueToParameter(double dValue) {
     } else if (dValue < 1.0) {
         // db + linear overlay to reach
         // m_minDB = 0
-        // 0 dB = m_neutralParameter
+        // 0 dB = m_dNeutralParameter
         double overlay = m_offset * (1 - dValue);
         if (m_minDB) {
-            dParam = (ratio2db(dValue + overlay) - m_minDB) / m_minDB * m_neutralParameter * -1;
+            dParam = (ratio2db(dValue + overlay) - m_minDB) / m_minDB * m_dNeutralParameter * -1;
```

Comment:
> Yeah I noticed that, but it happens that for ControlAudioTaperPotBehavior, it makes sense for m_dNeutralParameter and m_neutralParameter to be the same, because we definitely want the parameter that hides the ring completely to be the same that produces 0db gain.

A audio taper goes from -∞ dB ... 0 dB where -∞ dB is silence. In some cases the silent parameter should be the scale start point. For a EQ knob which has -12 ... + 12 dB the scale start is actually at 0 dB